dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReaderTest.java

```diff
@@ -200,11 +210,9 @@ private void checkSnpOrientation(List<SubSnpCoreFields> readSnps, Long snpId, Or
     @Test
     public void testQueryWithDifferentRelease() throws Exception {
         int dbsnpBuild = 130;
+        exception.expect(ItemStreamException.class);
```

Comment:
Is this a Spring batch exception?

Reply:
yes, that is thrown when either the reader, the processor or the writer fail and throw. That inner exception is wrapped in spring's ItemStreamException.